.github/workflows/docs-push.yml

```diff
@@ -19,11 +19,11 @@ jobs:
     name: Build Ansible Docs
     uses: ansible-community/github-docs-build/.github/workflows/_shared-docs-build-push.yml@main
     with:
-      init-lenient: false
+      init-lenient: true
```

Comment:
Generally we want this to be strict, so that we don't introduce warnings into the docs build.

I think with https://github.com/ansible-community/github-docs-build/pull/52 (now merged) you won't need to do this and uncomment below?